app/registries/foreman/plugin.rb

```diff
@@ -21,20 +21,21 @@
 require_dependency 'foreman/plugin/medium_providers_registry'
 require_dependency 'foreman/plugin/fact_importer_registry'
 
-module Foreman #:nodoc:
+#
+module Foreman
```

Comment:
I'm not sure how to convince yard that the above is a license and not a code comment. Suggestions welcome.